M2/Macaulay2/packages/Macaulay2Doc/ov_strings.m2

```diff
@@ -360,53 +360,69 @@ document {
      }
 
 undocumented {(format, Sequence)}
-document {
-     Key => {
-	  format,
-	 (format, CC),
-	 (format, RR),
-	 (format, String)},
-     Headline => "format a string or a real number",
-     SYNOPSIS (
-     	  Usage => "format(s)",
-	  Inputs => {"s"},
-	  Outputs => {{"a string obtained from ", TT "s", " by inserting 
-		    escape sequences, thereby preparing it for printing in a form suitable
-		    for reading in again"}},
-	  EXAMPLE lines ///
-	  s = "a\"b\"c"
-	  t = format s
-	  u = value t
-	  u == s
-	  ///
-	  ),
-     SYNOPSIS (
-     	  Usage => "format(s,k,l,t,e,x)",
-	  Inputs => {
-	       "s" => ZZ => {"the maximum number of significant decimal digits (default: ", TO "printingPrecision", ").
-		    The special value ", TT "0", " imposes no limit." },
-	       "k" => ZZ => {"how far to the right of the decimal point to go, at most (default: ", TO "printingAccuracy", ").
-		    The special value ", TT "-1", " imposes no limit."},
-	       "l" => ZZ => {"maximum number of leading zeroes (default: ", TO "printingLeadLimit", ")"},
-	       "t" => ZZ => {"maximum number of trailing zeroes (default: ", TO "printingTrailLimit", ")"},
-	       "e" => String => {"the separator between the mantissa and the exponent (default: ", TO "printingSeparator", ")"},
-	       "x" => RR => "the number to be converted to a string"
-	       },
-	  Outputs => {
-	       String => {"the decimal representation of the number ", TT "x", ", prepared according to the parameters above"}
-	       },
-	  EXAMPLE lines ///
-	      format(10,1/3000.)
-	      format(10,6,1/3000.)
-	      format(10,6,2,1/3000.)
-	      format(10,300000.)
-	      format(10,-1,10,5,300000.)
-	      format(10,-1,10,4,300000.)
-	      format(10,-1,10,4,"E",300000.)
-	  ///
-	  ),
-     SeeAlso => {toExternalString}
-     }
+
+doc ///
```

Comment:
Could you move this to `functions/format-doc.m2`? No need to run the tests.

Reply:
Will do!